ja_JP/LC_MESSAGES/blockSettings.po

```diff
@@ -1,29 +1,29 @@
 msgid "LOGIN_REQUIRED_TO_MANAGE_BLOCKS"
-msgstr "ブロックされたユーザーを管理するには、ログインする必要があります。"
+msgstr "ブロックしたユーザーを管理するには、ログインする必要があります。"
 
 msgid "NO_USERS_BLOCKED"
-msgstr "ブロックされたユーザーはいません。"
+msgstr "ブロックしたユーザーはいません。"
 
 msgid "BLOCKED_USERS"
-msgstr "ブロックされたユーザー"
+msgstr "ブロックしたユーザー"
 
 msgid "BLOCK_USER"
 msgstr "ブロックしたユーザー"
 
 msgid "COMMENT_BLOCK_IS_MUTUAL"
-msgstr "このユーザーをブロックすると、お互いの作品にコメントすることができなくなります。"
+msgstr "ユーザーをブロックすると、お互いに作品にコメントすることができなくなります。"
 
 msgid "USER_HAS_BEEN_BLOCKED"
 msgstr "このユーザーはすでにブロックしています。"
 
 msgid "IF_YOU_WISH_TO_UNBLOCK"
-msgstr "ブロックを解除するには、マイルームに行きブロックしたユーザーのメニューを開いてブロックを解除することができます。"
+msgstr "マイルームにある "ブロックしたユーザー" を開くことでブロックを解除することができます。"
```

Comment:
The double-quote breaks the string.

We will want to either escape it (`\"`) or change it to a single quote (`'`).

Reply:
Suggested:

```
 msgstr "マイルームにある \"ブロックしたユーザー\" を開くことでブロックを解除することができます。"
```